libraries/WiFi/src/WiFiGeneric.cpp

```diff
@@ -664,15 +664,6 @@ bool wifiLowLevelInit(bool persistent){
 
         wifi_init_config_t cfg = WIFI_INIT_CONFIG_DEFAULT();
 
-	if(!WiFiGenericClass::useStaticBuffers()) {
-	    cfg.static_tx_buf_num = 0;
-            cfg.dynamic_tx_buf_num = 32;
-	    cfg.tx_buf_type = 1;
-            cfg.cache_tx_buf_num = 1;  // can't be zero!
-	    cfg.static_rx_buf_num = 4;
-            cfg.dynamic_rx_buf_num = 32;
-        }
-
```

Comment:
I reject this PR and changes.... as explained in 
https://github.com/espressif/esp-idf/issues/8992#issuecomment-1197443125
https://github.com/espressif/esp-idf/issues/8992#issuecomment-1197446232